resources/js/src/app/components/wishList/WishList.js

```diff
@@ -3,15 +3,32 @@ const NotificationService = require("../../services/NotificationService");
 import TranslationService from "../../services/TranslationService";
 import Vue from "vue";
 import { mapState, mapActions } from "vuex";
+import WishListItem from "./WishListItem";
 
 Vue.component("wish-list", {
+    components:
+    {
+        WishListItem
+    },
 
-    delimiters: ["${", "}"],
-
-    props: [
-        "template",
-        "initIds"
-    ],
+    props:
+    {
+        template:
+        {
+            type: String,
+            default: "#vue-wish-list"
+        },
+        initIds:
```

Comment:
Wenn wir die Wunschliste in Zukunft überall erlauben, können wir die IDs nicht mehr als Parameter übergeben, da sie sonst im ShopBooster gecached werden würden.